helm/hpcc/values.yaml

```diff
@@ -510,7 +510,7 @@ roxie:
     numThreads: 30
     visibility: local
   ## replicas indicates the number of replicas per channel
-  replicas: 2  
```

Comment:
I don't think this change should have been checked in

Reply:
Sorry, it is for testing. I will resume the original settings